packages/astro/test/server/index.server.test.ts

```diff
@@ -1,4 +1,6 @@
 import sentryAstro from '../../src/index.server';
+import { describe, expect, it } from 'vitest';
```

Comment:
let's make the `vitest` import the first import on the file

Reply:
Oh, I thought linting would sort this all out!

Reply:
Ah, this is because since Biome was added, import order has not been enforced. I looked to get this working again and it impacts hundreds of files